metrics/rust-judged-behavior-panel/AUDIT-3198.md

```
@@ -0,0 +1,114 @@
+# Audit: #3198 acceptance against current main
+
+Audit authority: #3297. Audited main: `62a535db` (#3321 squash).
+Landed authority: #3246 (squash `2125d6ad`), #3251 (squash `93987f2b`),
+#3266/#3269 (squashes `15f9c679`, `62c2cdd8`), #3272 (squash `0446c60d`).
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)